bin/fzf-preview.sh

```diff
@@ -57,15 +57,15 @@ elif ! [[ $KITTY_WINDOW_ID ]] && (( FZF_PREVIEW_TOP + FZF_PREVIEW_LINES == $(stt
   dim=${FZF_PREVIEW_COLUMNS}x$((FZF_PREVIEW_LINES - 1))
 fi
 
-# 1. Use kitty icat on kitty terminal
-if [[ $KITTY_WINDOW_ID ]]; then
+# 1. Use icat (from Kitty) if kitten is installed
+if command -v kitten > /dev/null; then
```

Comment:
No, we should still check the environment variable. I have kitty installed so kitten is in the $PATH. So this passes regardless I use kitty or not.

Reply:
231876a